File: samples/wordnik-api-java/src/main/java/com/wordnik/client/model/WordFrequency.java
package com.wordnik.client.model;

public class WordFrequency {
  private long count = 0L;
  private String wordstring = null;
  public long getCount() {
    return count;
  }
  public void setCount(long count) {
    this.count = count;
  }

  public String getWordstring() {
    return wordstring;
  }
  public void setWordstring(String wordstring) {
    this.wordstring = wordstring;
  }

  @Override
  public String toString()  {
    StringBuilder sb = new StringBuilder();
    sb.append("class WordFrequency {\n");
    sb.append("  count: ").append(count).append("\n");
    sb.append("  wordstring: ").append(wordstring).append("\n");
    sb.append("}\n");
    return sb.toString();
  }
}

File: samples/wordnik-api-scala/src/main/scala/com/wordnik/client/model/WordFrequency.scala
package com.wordnik.client.model

import scala.reflect.BeanProperty

class WordFrequency {
  @BeanProperty var count: Long = 0L
  @BeanProperty var wordstring: String = _
  override def toString: String = {
    val sb = new StringBuilder
    sb.append("class WordFrequency {\n")
    sb.append("  count: ").append(count).append("\n")
    sb.append("  wordstring: ").append(wordstring).append("\n")
    sb.append("}\n")
    sb.toString
  }
}
